protocol/src/main/protobuf/Transactor.proto

```diff
@@ -14,14 +14,34 @@ message UpdateRequest {
     bytes chainCellCbor = 1;
 }
 
+message JournalStreamRequest {
+    MultihashReference lastJournalBlock = 1;
+}
+
+message ChainUpdateDescription {
+    MultihashReference canonicalRef = 1;
+    MultihashReference chainHeadRef = 2;
+}
+
+message JournalEvent {
+    oneof event {
+        MultihashReference canonicalInserted = 2;
+        ChainUpdateDescription chainUpdated = 3;
+        MultihashReference journalBlockPublished = 4;
+    }
+}
+
 service TransactorService {
     rpc InsertCanonical(InsertRequest) returns
         (MultihashReference);
 
     rpc UpdateChain(UpdateRequest) returns
-        (MultihashReference);
+        (ChainUpdateDescription);
```

Comment:
I'm not sure I love the "switch two words around and add either Request or Description" method of deriving these names